fix: Standardize width of footer

sachinchauhan2889 opened the pull request:
<!-- 
(Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.)
-->

<!-- Add the issue number that is fixed by this PR (In the form Fixes #123) -->

Fixes #5511 

#### Short description of what this resolves:
On the public event pages the sidebar spacing is different to the footer left and right spacing. This PR matched the left and right spacing of footer with sidebar and social links widget of public event page.

The footer is having a grid of 4 column which is wrapped in a center aligned container.
I have removed the container and created a grid with 3 columns. the 2 side column are empty(to match space with side bar of public event page and right side twitter widget). the center column is a nested grid which contain all footer data(grid of footer and all four columns.).
now footer is matched with the public event page.

#### Changes proposed in this pull request:

- matched the left and right spacing of footer with public event page side bar.
#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [X] My branch is up-to-date with the Upstream `development` branch.
- [X] The acceptance, integration, unit tests and linter pass locally with my changes <!-- use `ember test` to run all the tests -->
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

__screenshots__

__before__
![1111111111111111111111](https://user-images.githubusercontent.com/72552281/98394913-74f4c380-2081-11eb-9ab5-24c9dea89908.png)

__after__
![5511](https://user-images.githubusercontent.com/72552281/98394942-80e08580-2081-11eb-81f0-cb8ff2888026.png)

sachinchauhan2889 commented:
The footer is having a grid of 4 column which is wrapped in a center aligned container.
I have removed the container and created a grid with 3 columns. the 2 side column are empty(to match space with side bar of public event page and right side twitter widget).  the center column is a nested grid which contain all footer data(grid of footer and all four columns.).
now footer is matched with the public event page.

Raunaksingh100 commented on app/templates/components/footer-main.hbs:
```diff
@@ -1,127 +1,130 @@
   </div>
-</div>
+  <div class="three wide column"></div>
+</div>
```
add a new line

iamareebjamal replied:
```suggestion
</div>

```

Raunaksingh100 reviewed (changes requested):
add a new line

mariobehling reviewed (approved):
Works for me.

iamareebjamal reviewed (changes requested):
Why is there a need of such drastic structural changes? Please make minimum changes necessary

sachinchauhan2889 commented:
@iamareebjamal sir, please have a look at changes done by me. i have done minimum changes needed. I have removed the outer container and replace it with a Semantic UI Grid. 
In github it is showing i have changed 117 lines but actually i have changed 2 line and add 3 lines. i have screenshot of VS Code diaplaying changes done by me. please have a look at this.

![5511 chat](https://user-images.githubusercontent.com/72552281/98431083-671e5d00-20d8-11eb-9d52-5bf912347a23.png)

sachinchauhan2889 commented:
@iamareebjamal sir, have a look at this. There is no such drastic change in structure. Only outer centered aligner container is replaced with semantic ui grid and for matching left side bar and right twitter widget, 2 empty columns are added.
I think it is good approach for this problem.
Sir if you told to change, i will think for another solution.
